src/WidgetBase.ts

```diff
@@ -284,7 +285,14 @@ export class WidgetBase<P = WidgetProperties, C extends DNode = DNode> extends E
 	}
 
 	private _setNode(element: Element, properties: VNodeProperties): void {
-		this._nodeMap.set(String(properties.key), <HTMLElement> element);
+		const key = String(properties.key);
```

Comment:
Being reported as uncovered in tests.

Reply:
I'm seeing a "50x" next to that line

Reply:
It is because the CI isn't passing and therefore the code coverage isn't being uploaded.